src/modules/uniplayer/Search/SearchResults.js

```diff
@@ -11,11 +11,12 @@ const StyledSearchResults = styled.div`
 	max-height: 32rem;
 	position: absolute;
 	width: 100%;
+	z-index: 1;
 `;
 
-export default function SearchResults({results, enqueue, clearSearch}) {
+export default function SearchResults({results, enqueue, clearSearch, isSearchOpen}) {
 	return (
-		<StyledSearchResults onClick={clearSearch}>
+		<StyledSearchResults onClick={clearSearch} className={isSearchOpen ? 'large' : ''}>
```

Comment:
Don't need this class, and in general don't put classes on styled components, do 100% of their styling in the styled component.

That being said, we do need to put a class on this one to z-index.